Update stopwatch.js issue #1 Worked upon the Error stated

vivekBoii opened the pull request:
This pr is best because it taught me so many thing a tried many different appoached and able to decode how javascript works 

**thanks you sir**

vivekBoii committed:
Update stopwatch.js issue #1 Worked upon the Error stated

this pr is best because it taught me so many thing a tried many different appoached and able to decode how javascript works
thanks you sir

swagatmitra-b commented:
Thank you for the kind words. The restart functionality is working as expected but I'm afraid the reset button is not. Please look into it. Also, please change the base branch to "hacktoberfest"

vivekBoii committed:
Update stopwatch.js removed the faulty nature of reset button improved more on run button

Thank You for pointing my mistakes Highly appreciated Sir looking forward to work with you in future

swagatmitra-b commented:
Good job on the fix! But please change the base of your target branch to "hacktoberfest" and not "main"
